### ISSUE TITLE
Issue with uploading

### ISSUE THREAD

**Issue by ghost** [astrada/google-drive-ocamlfuse#713]
Hi,

I have around 1TB of data I need to upload. Whenever I mount the directory containing the files it does not start uploading. Is the only solution having a separate empty local directory and moving the files between the two directories? I thought it would work mounting the directory with all the files and it would start uploading automatically.

**Comment by astrada** [comment]
No, sorry, this isn't a sync app. For your use case, you should probably check [rclone](https://github.com/rclone/rclone) or [drive](https://github.com/odeke-em/drive).
